app/assets/javascripts/tabs.js

```diff
@@ -1,7 +1,7 @@
 $( document ).ready(function() {
   const url = new URL(window.location);
   $(".lux-tabs-container li").removeClass( "active" );
-  if(!url.href.includes("_") && !url.hash){
+  if(!url.href.includes("_") && !url.hash && $("#component-summary").length == 0){
```

Comment:
Why do we test for these things? Do we know? Can we comment what is going on here or move these conditionals to a named method that sheds light on this situation?